# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/reftest.list
# Commit: 51ba9e64ff0a
# Full Hash: 51ba9e64ff0a4071cdd57dbf24110606faad781e
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-27 15:58:57
# Description:
#   Bug 1771388 - Ensure backdrop capture has same clip as backdrop render r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147482
# ==============================================================================

diff -r 5e2d37a71968 -r 51ba9e64ff0a gfx/wr/wrench/reftests/filters/reftest.list
--- a/gfx/wr/wrench/reftests/filters/reftest.list	Fri May 27 04:32:40 2022 +0000
+++ b/gfx/wr/wrench/reftests/filters/reftest.list	Fri May 27 05:38:29 2022 +0000
@@ -73,6 +73,7 @@
 #platform(linux,mac) == backdrop-filter-drop-shadow.yaml backdrop-filter-drop-shadow.png
 == backdrop-filter-blur-edge-mode.yaml backdrop-filter-blur-edge-mode-ref.yaml
 == backdrop-filter-half-opacity.yaml backdrop-filter-half-opacity-ref.yaml
+== backdrop-filter-culled-filter.yaml backdrop-filter-culled-filter-ref.yaml
 platform(linux,mac) == backdrop-filter-transformed-filter.yaml backdrop-filter-transformed-filter.png
 platform(linux,max) == svg-filter-offset.yaml svg-filter-offset-ref.yaml
 skip_on(android,device) == fuzzy(1,100) svg-filter-composite.yaml svg-filter-composite-ref.yaml
